app/views/coc/divorce_select_year.scala.html

```diff
@@ -49,8 +49,10 @@ <h1 class="heading-xlarge">@messages("pages.divorce.title")</h1>
             DivorceSelectYearForm.DateOfDivorce,
             Html(""),
             None,
+            '_legend -> Messages("pages.divorce.title"),
```

Comment:
can we use the implicit messages here
```suggestion
            '_legend -> messages("pages.divorce.title"),
```